docs/contents/index.md

```diff
@@ -89,23 +90,25 @@ releases will be created when there is an updated component version, a required
 update in the Amazon Linux 2 base image, or a change required in the build
 toolchain (ex: a Go version update).
 
-Components such as CNI, etcd, aws-iam-authenticator, and Kubernetes have
-release artifacts that are not delivered as container images, and are available
-as compressed tar archives and binaries. sha256 and sha512 sum files are
-provided in release manifests and are available as files for download (ex:
-`https://distro.eks.amazonaws.com/..../kubectl.sha256`)
+There are components of Kubernetes, CNI, etcd, and aws-iam-authenticator
+that are not delivered as container images. These components are available as
+compressed tar archive and executable files. The files have associated sha256
+and sha512 sum files provided in release manifests. They are available for download
+with names similar to the file (ex: `https://distro.eks.amazonaws.com/..../kubectl.sha256`)
 
 A list of all the components and assets that make up a release including URIs
 to all the compressed archives, binaries, and container images are available in
```

Comment:
nit:  A list of all the components and assets... ~~are~~`is` available in the release manifests.

Reply:
Done